Makefile

```diff
@@ -312,20 +313,17 @@ schemas_format: schemas_docker_build  ## Format schemas source tree
 
 schemas_lint: schemas_docker_build  ## Lint schemas source tree
 	$(SCHEMAS_RUN) "$(SCHEMAS_BLACK)&&$(SCHEMAS_RUFF)&&$(SCHEMAS_DIFF_PYDANTIC)&&$(SCHEMAS_TEST)"
-schemas_check: schemas_lint
 
-schemas_dist_pypi: schemas_docker_build
-	$(SCHEMAS_RUN) "$(SCHEMAS_DIST_PYPI)"
+schemas_check: schemas_lint
 
-schemas_dist_npm: schemas_docker_build schemas_dist_pypi
-	$(SCHEMAS_RUN) "$(SCHEMAS_DIST_NPM)"
+schemas_generate: schemas_docker_build
+	$(SCHEMAS_RUN) "$(SCHEMAS_GENERATE)"
 
-schemas_dist: schemas_docker_build schemas_dist_pypi schemas_dist_npm
 
-schemas_deploy_pypi: schemas_docker_build
+schemas_deploy_pypi: schemas_build
 	$(SCHEMAS_RUN) "$(SCHEMAS_DEPLOY_PYPI)"
 
-schemas_deploy_npm: schemas_docker_build
+schemas_deploy_npm: schemas_build
```

Comment:
Is it always desirable to build before deploy? And if so, is there any particular advantage to enforcing it?

Most importantly, if we do this, we need to update the CI config -- currently it runs `make schemas_dist` and then the deploy steps.

Reply:
Yes it is desired. We *need* the build step to run to copy the schemas into the python package.